WcaOnRails/app/models/event.rb

```diff
@@ -54,4 +54,12 @@ def fewest_moves?
   def multiple_blindfolded?
     self.id == "333mbf" || self.id == "333mbo"
   end
+
+  def serializable_hash(options = nil)
```

Comment:
Ah, whoops. This is unused right now, and is needed to expose event information to javascript in my upcoming UI changes that will use this API.

I plan to add a feature spec for the relevant javascript code once things have settled down there. I can either leave this here if you're okay with it, or do the git-foo required to move it to a later PR.